src/index.js

```diff
@@ -40,6 +56,14 @@ const _init = (options) => {
   }
 
   snippet(options);
+
+  if (options.devMode) {
+    event('FullStory Dev Mode', {
```

Comment:
What if the event were part of a class of events, i.e. `FullStory SDK`.  Then you could have future SDK messages similar grouped logically - similar to how FullStory groups API event/customerVar errors.

Reply:
@van-fs Are you thinking of some kind of taxonomy that we apply the event name?

We also have the option of adding "FullStorySDK" to the (undocumented) `source` parameter of `FS.event`. I've had this idea in mind for a while now but want to approach it with care (b/c I think we'd also want to provide the ability for technology partners to change the source of `FS.event` as well).

Reply:
I'm good with the current implementation.  If we started to get more error checking and messaging coming out of the SDK, engineering might just want to have a segment that looks at `production.mysite.com` and any `FullStory SDK` messages, which could indicate a possible issue.  This would be the alternative to multiple event names.